IntegrationTests/IRVMR/script/makeProject.tcl

```diff
@@ -0,0 +1,46 @@
+# Create Vivado project, with user HDL files & IP.
+# Run this in IntegrationTests/xyz/script/ 
+
+# Create project
+set projName "Work"
+set FPGA "xcvu7p-flvb2104-1-e"
+create_project -force ${projName} ./${projName} -part $FPGA
+set_property target_language VHDL [current_project]
+
+# Rebuild user HLS IP repos index before adding any source files
+set_property ip_repo_paths "../../../project/"  [get_filesets sources_1]
+update_ip_catalog -rebuild
+
+# Create .xci files for user HLS IP
+# Chain contains two IRs and one VMR
+# Note: currently uses the same IP core for the two IRs. OK for this minimal example but probably not in general.
+create_ip -name InputRouterTop -module_name IR_PS10G_3_A -vendor xilinx.com -library hls -version 1.0
+create_ip -name InputRouterTop -module_name IR_negPS10G_3_A -vendor xilinx.com -library hls -version 1.0
+create_ip -name VMRouterTop -module_name VMR_L2PHIA -vendor xilinx.com -library hls -version 1.0
+
+# Provide name of top-level HDL (without .vhd extension).
+#set topLevelHDL "SectorProcessor"
+set topLevelHDL "SectorProcessorFull"
+
+# Add HDL for algo
+add_files -fileset sources_1 [glob ../hdl/SectorProcessor*.vhd]
+add_files -fileset sources_1 [glob ../hdl/memUtil_pkg.vhd]
+add_files -fileset sources_1 [glob ../../common/hdl/*.vhd]
+
+# Add HDL for TB
+#add_files -fileset sim_1 [glob ../tb/tb_tf_top.vhd]
+
+# Set 'sim_1' fileset properties
+set_property file_type {VHDL 2008} [get_files -filter {FILE_TYPE == VHDL}]
+set_property top -value ${topLevelHDL} -objects [get_filesets sources_1]
+#set_property top -value "tb_tf_top" -objects [get_filesets sim_1]
+set_property xsim.simulate.runtime -value "50us" -objects  [get_filesets sim_1]
+
+update_compile_order -fileset sources_1 
+
+# set mode to out_of_context as the VMR has too many input/output ports
```

Comment:
As common/script/synth.tcl already sets the mode to out_of_context, do you really need it in makeProject.tcl too? It shouldn't be needed for the simulation.

Reply:
It's because I like to run the synthesis in the GUI for some reason, I'll remove it.